k8sutils/pkg/container/container.go

```diff
@@ -1,111 +1,11 @@
 package container
 
 import (
-	"errors"
-	"strings"
-
 	v1 "k8s.io/api/core/v1"
 
 	odigosv1 "github.com/odigos-io/odigos/api/odigos/v1alpha1"
-	"github.com/odigos-io/odigos/common"
-)
-
-var (
-	ErrDeviceNotDetected     = errors.New("device not detected")
-	ErrContainerNotInPodSpec = errors.New("container not found in pod spec")
 )
 
-func LanguageAndSdk(pod *v1.Pod, containerName string, distroName string) (common.ProgrammingLanguage, common.OtelSdk, error) {
```

Comment:
removing this function was the main motivation. many references to `common.OtelSdk` which we unlink 🎉